docs/whirlpool/package.json

```diff
@@ -5,11 +5,14 @@
   "scripts": {
     "build": "docusaurus build --out-dir dist",
     "start": "docusaurus start",
+    "swizzle": "docusaurus swizzle",
     "clean": "rimraf dist .docusaurus"
   },
   "dependencies": {
     "@docusaurus/core": "3.5.2",
     "@docusaurus/preset-classic": "3.5.2",
+    "@emotion/react": "^11.13.3",
```

Comment:
Does docusaurus support tw? If so we might want to use that instead of `@emotion`